docs/pages/7_0_0_release_notes.md

```diff
@@ -123,16 +165,21 @@ We are shipping the following rules:
   or `hashCode` overridden, but not both. This leads to unexpected behavior once instances of such classes
   are used in collections (Lists, HashMaps, ...).
 
-Contributors: [@jborgers](https://github.com/jborgers), [@stokpop](https://github.com/stokpop)
+Contributors: [Jeroen Borgers](https://github.com/jborgers) (@jborgers),
+  [Peter Paul Bakker](https://github.com/stokpop) (@stokpop)
 
 #### XPath 3.1 support
 
-Support for XPath versions 1.0, 1.0-compatibility was removed, support for XPath 2.0 is deprecated. The default (and only) supported XPath version is now XPath 3.1. This version of the XPath language is mostly identical to XPath 2.0. Notable changes:
- * The deprecated support for sequence-valued attributes is removed. Sequence-valued properties are still supported.
- * Refer to [the Saxonica documentation](https://www.saxonica.com/html/documentation/expressions/xpath31new.html) for an introduction to new features in XPath 3.1.
+Support for XPath versions 1.0, 1.0-compatibility was removed, support for XPath 2.0 is deprecated. The default
+(and only) supported XPath version is now XPath 3.1. This version of the XPath language is mostly identical to
+XPath 2.0.
 
+Notable changes:
+* The deprecated support for sequence-valued attributes is removed. Sequence-valued properties are still supported.
+* Refer to [the Saxonica documentation](https://www.saxonica.com/html/documentation/expressions/xpath31new.html) for
+  an introduction to new features in XPath 3.1.
 
-#### Node stream API
+#### Node stream API for AST traversal
```

Comment:
I'd probably move this section a little further down, next to API, moving from the more user-focused changes and towards the more technical ones.